scapy/packet.py

```diff
@@ -786,7 +787,7 @@ def do_dissect(self, s):
             if f.islist or f.holds_packets or f.ismutable:
                 self.raw_packet_cache_fields[f.name] = f.do_copy(fval)
             self.fields[f.name] = fval
-        assert(_raw.endswith(raw(s)))
+        # assert(_raw.endswith(raw(s)))
```

Comment:
Why is this line commented?

Reply:
It’s not really needed, as it is always correct. I guess it could used when debugging the function so I left it as a comment :/

Reply:
Done